backup_push_handler.go

```diff
@@ -21,7 +21,7 @@ func getDeltaConfig() (maxDeltas int, fromFull bool) {
 	origin, hasOrigin := os.LookupEnv("WALG_DELTA_ORIGIN")
 	if hasOrigin {
 		switch origin {
-		case "LATEST":
+		case LatestString:
```

Comment:
Is this the only LATEST literal?

Reply:
No, but other occurrences are inside of user message strings.